core-primitives/node-api/metadata/src/metadata_mocks.rs

```diff
@@ -54,14 +60,26 @@ impl NodeMetadataMock {
 }
 
 impl TeerexCallIndexes for NodeMetadataMock {
-	fn register_enclave_call_indexes(&self) -> Result<[u8; 2]> {
-		Ok([self.teerex_module, self.register_enclave])
+	fn register_ias_enclave_call_indexes(&self) -> Result<[u8; 2]> {
```

Comment:
How about making the [u8; 2] into its own type alias as well to reduce code duplication?

Reply:
This could make sense, but I would prefer to do that in a separate PR as it would also affect other code such as the teeracle. What do you think?

Reply:
Then I think that it should be done in a separate PR :)

Reply:
Opened an issue for it: https://github.com/integritee-network/worker/issues/1151